src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -3815,6 +3818,80 @@
     public static final int TEXT_BLOCK_LITERAL_END =
             GeneratedJavaTokenTypes.TEXT_BLOCK_LITERAL_END;
 
+    /**
+     * The {@code yield} keyword.  This element appears
+     * as part of a yield statement.
+     *
+     * @see #LITERAL_SWITCH
+     * @see #CASE_GROUP
+     * @see #SLIST
+     * @see #SWITCH_RULE
+     *
+     * @see <a href="https://docs.oracle.com/javase/specs/jls/se13/preview/switch-expressions.html">
+     * Java Language Specification, &sect;14.21</a>
+     *
+     * @since 8.36
+     **/
+    public static final int LITERAL_YIELD =
+            GeneratedJavaTokenTypes.LITERAL_yield;
+
+    /**
+     * A switch labeled rule has a switch label and either an expression,
+     * block, or throw statement. A switch labeled statement group has one
+     * or more switch labels and zero or more block statements. A switch label
+     * is either a case or default label.
```

Comment:
wording is very strange.
It looks like brief of grammar. I doubt user need it, we never did this for other tokens.

Can we simply do ? 
`Switch Expressions` as content.
all details on what is possible should be read in java specification(that might change in future), this file is not good place for this.

Reply:
Fixed.